client/jetpack-cloud/sections/settings/advanced-credentials/host-info.ts

```diff
@@ -79,64 +97,102 @@ export const topHosts: Host[] = [
-					translate(
-						'FTP (File Transfer Protocol): the original standard for transferring files between servers.'
-					).toString(),
-					translate(
-						'SFTP (Secure File Transfer Protocol): is like FTP, but adds a layer of security (SSH encryption).'
-					).toString(),
-					translate(
-						'SFTP/SSH is the preferred method to choose. Both methods are supported by Bluehost.'
-					).toString(),
+		inline: {
+			credentialType: {
+				ftp: [
+					{
+						type: InfoTypes.UnorderedList,
+						items: [
+							translate(
+								'FTP (File Transfer Protocol): the original standard for transferring files between servers.'
+							).toString(),
+							translate(
+								'SFTP (Secure File Transfer Protocol): is like FTP, but adds a layer of security (SSH encryption).'
+							).toString(),
+							translate(
+								'SFTP/SSH is the preferred method to choose. Both methods are supported by Bluehost.'
+							).toString(),
+						],
+					},
+
+					{
+						type: InfoTypes.Line,
+					},
+					{
+						type: InfoTypes.Link,
+						text: translate( 'Read more' ).toString(),
+						link: 'https://my.bluehost.com/cgi/help/ftpaccounts',
+					},
+				],
+				sftp: [
+					{
+						type: InfoTypes.UnorderedList,
+						items: [
+							translate(
+								'FTP (File Transfer Protocol): the original standard for transferring files between servers.'
+							).toString(),
+							translate(
+								'SFTP (Secure File Transfer Protocol): is like FTP, but adds a layer of security (SSH encryption).'
+							).toString(),
+							translate(
+								'SFTP/SSH is the preferred method to choose. Both methods are supported by Bluehost.'
+							).toString(),
+						],
+					},
+
+					{
+						type: InfoTypes.Line,
+					},
+					{
+						type: InfoTypes.Link,
+						text: translate( 'Read more' ).toString(),
+						link: 'https://www.bluehost.com/help/article/ssh-access',
+					},
 				],
 			},
-			{
-				type: InfoTypes.Line,
-			},
-			{
-				type: InfoTypes.Link,
-				text: translate( 'Read more' ).toString(),
-				link: 'https://my.bluehost.com/cgi/help/ftpaccounts',
-			},
-		],
-		serverAddress: [
-			{
-				type: InfoTypes.Text,
-				text: translate(
-					'Your Domain Name or server IP address. Both are available from the Bluehost cPanel.'
-				).toString(),
-			},
-			{
-				type: InfoTypes.Line,
-			},
-			{
-				type: InfoTypes.Link,
-				text: translate( 'Vist my Bluehost cPanel' ).toString(),
-				link: 'https://my.bluehost.com/cgi-bin/cplogin',
-			},
-		],
-		portNumber: [
-			{
-				type: InfoTypes.Text,
-				text: translate( 'Enter port 21 for Bluehost’s FTP service.' ).toString(),
+			serverAddress: [
+				{
+					type: InfoTypes.Text,
+					text: translate(
+						'Your Domain Name or server IP address. Both are available from the Bluehost cPanel.'
+					).toString(),
+				},
+				{
+					type: InfoTypes.Line,
+				},
+				{
+					type: InfoTypes.Link,
+					text: translate( 'Visit my Bluehost cPanel' ).toString(),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **12** times:
`translate( 'Vist my Bluehost cPanel' )` *ES Score: 10*